### DISCUSSION TITLE
Support for `//` paths?

### DISCUSSION THREAD

**Discussion by gpl** [helmfile/helmfile#337] [Ideas]
We would like to be able to specify local chart paths by the repo they're in, for example, it would be useful if we could specify //path/to/chart instead of ../../../chart

**Comment by yxxhero** [comment]
@gpl Please show a example. Thanks very much.

**Comment by gpl** [comment]
We have a setup with deeply nested helmfiles that looks something like this:

```

repo/.git
repo/a/helmfile.yaml
repo/a/b/c/d/1/helmfile.yaml
repo/a/b/c/d/2/helmfile.yaml
repo/charts/some-app-1
repo/charts/some-app-2
```

```
repo/a/helmfile.yaml
helmfiles:
  - path: repo/a/b/c/d/1/helmfile.yaml
  - path: repo/a/b/c/d/2/helmfile.yaml
```

Right now, we have to do something like this:

> repo/a/b/c/d/2/helmfile.yaml
```
releases:
- name: some-app-1
  chart: ../../../../../charts/some-app/1
```
We would like to do something like this instead:

> repo/a/b/c/d/2/helmfile.yaml
```
releases:
- name: some-app-1
  chart: //charts/some-app/1
```

Right now the directory traversal can get a bit messy and it would be nice to be able to use the git repo as the root and use it as an anchor.
